plugins/net_plugin/net_plugin.cpp

```diff
@@ -772,16 +769,11 @@ namespace eosio {
       }
 
       void operator()( signed_block&& msg ) const {
-         // continue call to handle_message on connection strand
-         shared_ptr<signed_block> ptr = std::make_shared<signed_block>( std::move( msg ) );
-         c->handle_message( ptr );
+         EOS_ASSERT( false, plugin_config_exception, "operator()(signed_block&&) should call handle_message" );
```

Comment:
I don't understand the phrasing of this error message.  Or the one on the new line 776.

Reply:
Updated these all to say: `"Not implemented, call handle_message directly instead"`